tests/core/test_plan.py

```diff
@@ -1548,7 +1548,7 @@ def test_indirectly_modified_forward_only_model(make_snapshot, mocker: MockerFix
 
     assert updated_snapshot_a.change_category == SnapshotChangeCategory.BREAKING
     assert updated_snapshot_b.change_category == SnapshotChangeCategory.FORWARD_ONLY
-    assert updated_snapshot_c.change_category == SnapshotChangeCategory.INDIRECT_BREAKING
+    assert updated_snapshot_c.change_category == SnapshotChangeCategory.FORWARD_ONLY
```

Comment:
Let's also make test in which C points at both A and B, in which case, the C's category should indeed be `INDIRECT_BREAKING`.

Reply:
done